content/staging-area.md

```diff
@@ -161,61 +161,134 @@ We give two examples and the instructor can pick one or both:
 
 ## Staging area commands
 
-With the **staging area**, there is one middle point before commits.
+The staging area creates a layer between the working directory and the next
+commit. This adds some complexity but also adds more flexibility to selectively
+prepare commits since **you can modify and stage several times before committing**.
+
+**git add** stages/prepares for the next commit:
+```text
+                git add
+ [project*]  <------------  project*
+                               ^
+                               | modify with editor
+                               |
+  HEAD            .         project
+    |             .
+    |             .
+    |             .
+  HEAD~1          .
+    |             .
+    |             .
+    |             .
+                  .
+  .git            .     working directory
+(history)         .      (files we see)
+```
 
-```{figure} img/staging-basics.svg
-:alt: Staging basics
-:width: 100%
```

Comment:
I'm biased since I made that picture, but once I did stuff finally started clicking for me.
> There are three "locations" (working, staging, HEAD), and each pair has a command that shows the differences and updates one to look like the other.

Could this be moved to the bottom, or perhaps moved to reference.md ?  Maybe that text + the picture could be used as a "wrap up" at the bottom?

I'd agree that this should not be the first thing shown

Reply:
... though it could be updated for "git restore", but that could be written below saying "with new enough git this works"?

Reply:
I will put it back and update.